File: fixtures/scale/kotlin/small/src/textprinter/border/Frames.kt
package textprinter.border

fun asciiFrame(): Frame = Frame('+', '-', '|')

fun roundedFrame(): Frame = Frame('o', '─', '│')

fun doubleFrame(): Frame = Frame('╔', '═', '║')

fun frameNamed(name: String): Frame = when (name) {
    "rounded" -> roundedFrame()
    "double" -> doubleFrame()
    else -> asciiFrame()
}

File: fixtures/scale/dart/small/lib/border/frames.dart
import 'frame.dart';

const asciiFrame = Frame('+', '-', '|');
const roundedFrame = Frame('o', '-', '|');

Frame frameNamed(String name) {
  switch (name) {
    case 'rounded':
      return roundedFrame;
    default:
      return asciiFrame;
  }
}
